packages/components/src/notice-banner/style.scss

```diff
@@ -105,13 +105,36 @@
 }
 
 .notice-banner.is-info {
-	border-left-color: var(--black);
+	border-left-color: var(--jp-black);
 
 	.notice-banner__icon {
-		fill: var(--black);
+		fill: var(--jp-black);
+	}
+
+	.notice-banner__action-button,
+	.notice-banner__action-link {
+		cursor: grab;
```

Comment:
Is this cursor set to `grab` for some reason rather than `pointer`?

Reply:
I agree with Dognose; shouldn't this be `pointer`?

Reply:
Thanks for catching it @dognose24 & @jsnmoon, I was testing out something and left this by accident. Fixed in 27c08afba6